integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/mediatorconfig/fast_xslt/calltemplate_integration_fastxslt_sample750_synapse.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <definitions xmlns="http://ws.apache.org/ns/synapse">
-    <proxy name="StockQuoteProxy" startOnLoad="true">
+    <proxy name="StockQuoteProxyForFastXSLT-mediator-Test" startOnLoad="true" transports="http">
```

Comment:
Shall we rename the one that's used in normal xslt tests rather than this since it's the one that is being predeployed?